lib/msal-electron-proof-of-concept/test/appConfig/applicationConfiguration.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { expect } from 'chai';
+import * as Mocha from 'mocha';
+
+import { ApplicationConfiguration, buildApplicationConfiguration } from '../../src/index';
+
+describe('ApplicationConfiguration.ts', () => {
+    // Test URIs
+    const TEST_REDIRECT_URI = 'test://auth';
```

Comment:
>TEST_REDIRECT_URI [](start = 10, length = 17)

can these be camelcase instead?

Reply:
The JS/TS convention for constants values is all caps. I'm removing this line in another PR and passing the value to a separate file that will hold all test constants like we have in msal-core JS.